pkgs/development/python-modules/qdldl/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, cmake
+, pybind11
+, numpy
+, scipy
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "qdldl";
+  version = "0.1.5.post0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "c392c7427651d8b226423c7aba4a0f2338a1f38a4bbdabac6bc4afd8bc934f06";
+  };
+
+  dontUseCmakeConfigure = true;
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ pybind11 ];
+
+  propagatedBuildInputs = [
+    numpy
+    scipy
+  ];
+
+  pythonImportsCheck = [ "qdldl" ];
+
+  checkInputs = [ pytestCheckHook ];
+  dontUseSetuptoolsCheck = true;
+  # for nixos-20.03
```

Comment:
20.03 is EOL and no longer supported. We do not backport anything either so this should be removed.

Reply:
Ack, will fix. That was an artifact from copying it from my nur packages build where I prototype most of these updates.